packages/account/src/Components/icon-message-content/icon-message-content.tsx

```diff
@@ -16,14 +17,15 @@ const IconMessageContent = ({
     className,
     full_width,
     icon,
+    is_disabled = false,
     message,
     text,
 }: React.PropsWithChildren<TIconMessageContent>) => (
     <Div100vhContainer
         className={classNames('account-management__message-wrapper', {
             'account-management__message-wrapper-full-width': full_width,
         })}
-        is_disabled={isDesktop()}
+        is_disabled={is_disabled || isDesktop()}
```

Comment:
Could you please elaborate on this change?
if we want to adjust 'is_disabled' outside the component, anyway it is impossible to get this property falsy for desktop, right?

Reply:
`is_disabled` prop is used inside `Div100vhContainer` only to set a fixed height in for the div. 
so as per this change, `is_disabled` needs to be passed only if we need to prevent the component from setting a fixed height in `mobile screen only`.
i made this change because i needed to display a button below this component without empty space in between(so had to disable the height styling). if this is confusing, please let me know 😄

Reply:
maybe shall we rename it to `is_disabled_for_mobile `?

Reply:
@shaheer-deriv, sounds more understandable :) 
up to you 👍